package.json

```diff
@@ -25,11 +25,12 @@
     "prettier": "2.8.8",
     "prettier-plugin-svelte": "^2.3.0",
     "proper-lockfile": "^4.1.2",
-    "svelte": "^4.2.10",
+    "svelte": "4.2.19",
```

Comment:
We should define our best practices in here, as we always have a mix. Should we pin or use minor versions?

Reply:
I think we should pin - as it keeps the `yarn.lock` more consistent